modules/rules_swift/1.2.0/source.json

```diff
@@ -0,0 +1,5 @@
+{
+    "integrity": "sha256-Ue/a+F4E5RF03nbvVj8lVFHVpc0kxhrZAv7q2vxwRtk=",
+    "strip_prefix": "rules_swift.1.2.0",
```

Comment:
CI failures make me think this line should go.

Reply:
Done! Do you mind unblocking CI for me?